packages/devtools_app/lib/src/screens/memory/memory_protocol.dart

```diff
@@ -264,20 +266,19 @@ class MemoryTracker {
   /// events until the next HeapSample (tick) received see [_recalculate].
   EventSample pullClone(MemoryTimeline memoryTimeline, int time) {
     final pulledEvent = memoryTimeline.pullEventSample();
-    final extensionEvents = memoryTimeline.extensionEvents;
+    final ExtensionEvents? extensionEvents = memoryTimeline.extensionEvents;
```

Comment:
this type shouldn't need to be here because memoryTimeline.extensionEvents already has type ExtensionEvents?

Reply:
now, when memoryTimeline is migrated, yes